src/components/Awards.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import generateKey from '@tammiec/generatekey';
+
+const Awards = ({ data, ...props }) => {
+
+  const awards = data.map(award => (
+    <div key={generateKey()}>
```

Comment:
Be careful with generated keys can lead up into render inconsistencies if this component get many re-renders, the best practices it's to have an constant id for each element already in the array